Chirp/Pages/MorePage/MorePageViewController.swift

```diff
@@ -0,0 +1,115 @@
+//
+//  MorePageViewController.swift
+//  Chirp
+//
+//  Created by Siarhei Anoshka on 8.06.23.
+//
+
+import UIKit
+
+class MorePageViewController: UIViewController {
+
+    var cellNames: [String] = ["Push Notification", "Contributors", "Logout"]
```

Comment:
consider using enum

Reply:
Great comment here.
@stepharski if you can, it would be nice to have this as an enum instead of a string. 
It will also make it easier to identify the selected row and perform some operation instead of using raw strings.

```swift
enum CellName: String {
  case pushNotification = "Push Notification
 .....
}
```

Then in your cell for row you can get it as cellNames.rawValue